{"uid":"30f4a54a1b655cb6","name":"test_change_carcenter","fullName":"scripts.test02_change_carcenter.TestChangeCarcenter#test_change_carcenter","time":{"start":1699841706835,"stop":1699841706835,"duration":0},"status":"skipped","flaky":false,"newFailed":false,"newBroken":false,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"_xunit_setup_method_fixture_TestChangeCarcenter","time":{"start":1699841706835,"stop":1699841712941,"duration":6106},"status":"broken","statusMessage":"KeyboardInterrupt\n","statusTrace":"  File \"D:\\software\\Python310\\lib\\site-packages\\pluggy\\_callers.py\", line 77, in _multicall\n    res = hook_impl.function(*args)\n  File \"D:\\software\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 1123, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n  File \"D:\\software\\Python310\\lib\\site-packages\\_pytest\\fixtures.py\", line 895, in call_fixture_func\n    fixture_result = next(generator)\n  File \"D:\\software\\Python310\\lib\\site-packages\\_pytest\\python.py\", line 899, in xunit_setup_method_fixture\n    _call_with_optional_argument(func, method)\n  File \"D:\\software\\Python310\\lib\\site-packages\\_pytest\\python.py\", line 779, in _call_with_optional_argument\n    func()\n  File \"E:\\DM\\ShirleyPytest\\scripts\\test02_change_carcenter.py\", line 19, in setup_method\n    self.driver = init_driver()\n  File \"E:\\DM\\ShirleyPytest\\base\\base_driver.py\", line 16, in init_driver\n    driver = webdriver.Remote(\"http://127.0.0.1:4723\", caps)\n  File \"D:\\software\\Python310\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 257, in __init__\n    super().__init__(\n  File \"D:\\software\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 205, in __init__\n    self.start_session(capabilities)\n  File \"D:\\software\\Python310\\lib\\site-packages\\appium\\webdriver\\webdriver.py\", line 346, in start_session\n    response = self.execute(RemoteCommand.NEW_SESSION, w3c_caps)\n  File \"D:\\software\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py\", line 342, in execute\n    response = self.command_executor.execute(driver_command, params)\n  File \"D:\\software\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 297, in execute\n    return self._request(command_info[0], url, body=data)\n  File \"D:\\software\\Python310\\lib\\site-packages\\selenium\\webdriver\\remote\\remote_connection.py\", line 318, in _request\n    response = self._conn.request(method, url, body=body, headers=headers)\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\_request_methods.py\", line 118, in request\n    return self.request_encode_body(\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\_request_methods.py\", line 217, in request_encode_body\n    return self.urlopen(method, url, **extra_kw)\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\poolmanager.py\", line 443, in urlopen\n    response = conn.urlopen(method, u.request_uri, **kw)\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 874, in urlopen\n    return self.urlopen(\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 790, in urlopen\n    response = self._make_request(\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connectionpool.py\", line 496, in _make_request\n    conn.request(\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connection.py\", line 395, in request\n    self.endheaders()\n  File \"D:\\software\\Python310\\lib\\http\\client.py\", line 1278, in endheaders\n    self._send_output(message_body, encode_chunked=encode_chunked)\n  File \"D:\\software\\Python310\\lib\\http\\client.py\", line 1038, in _send_output\n    self.send(msg)\n  File \"D:\\software\\Python310\\lib\\http\\client.py\", line 976, in send\n    self.connect()\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connection.py\", line 243, in connect\n    self.sock = self._new_conn()\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\connection.py\", line 203, in _new_conn\n    sock = connection.create_connection(\n  File \"D:\\software\\Python310\\lib\\site-packages\\urllib3\\util\\connection.py\", line 73, in create_connection\n    sock.connect(sa)\n","steps":[],"attachments":[],"parameters":[],"shouldDisplayMessage":true,"attachmentsCount":0,"hasContent":true,"stepsCount":0}],"afterStages":[],"labels":[{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","categories":[],"tags":[]},"source":"30f4a54a1b655cb6.json","parameterValues":[]}